benchmark/dns/lookup.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common.js');
+const lookup = require('dns').lookup;
+
+const bench = common.createBenchmark(main, {
+  name: ['', '127.0.0.1', '::1'],
```

Comment:
Does it make sense to throw in `localhost` so that there's a hostname lookup since that probably takes a different path through the code?